xml/vt_scenarios.xml

```diff
@@ -0,0 +1,198 @@
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Better usage of resources which were not running at 100%
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Fewer servers locations needed
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Compute resources more efficiently: multiple workloads on the same server
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Simplification of the data center infrastructure
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Moving workload is easier to another host, this will avoid service downtime 
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Virtual Machine provisioning
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Multiple guest operating system will be able to run on 1 host
+    </para>
+   </listitem>
+  </itemizedlist>
+  <important>
+   <para>The server consolidation scenario needs special attention.</para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Maintenance windows should be carefully planned
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Storage is a key point to be able to support migration and growing disk usage
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Server capabilities must be checked to support the new workload
+     </para>
+    </listitem>
+   </itemizedlist>
+  </important>
+ </sect1>
+ <sect1>
+  <title>Isolation</title>
+  <para>
+   A guest operating system can be fully isolated from the host
+   running it. So if the virtual machine is corrupted, the host system is
+   not harmed. It also does not affect any other virtual machines. No data
+   is shared between VM.
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Secure Boot can be used for VM
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     KSM should be avoided
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     CPU can be assigned to a VM, and HT should be disabled to avoid security issue
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     VM should not share network / storage / network / Hardware
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Migration capabilities could be impacted in case of specific usage like PCI passthrough or NUMA
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Full Virtualization is generally less efficient if no PV/virtio drivers are used
+    </para>
+   </listitem>
+  </itemizedlist>
+<para>
+    AMD provides some specific features regarding Virtualization security.
+    <!-- link to the SEV doc? -->
+   </para>
+  <itemizedlist>
+   
+   <listitem>
+    <para>
+     AMD Secure Memory Encryption (SME):
+     Uses a single key to encrypt system memory. The key is generated by the AMD Secure Processor at boot. SME requires enablement in the system BIOS or operating system. When enabled in the BIOS, memory encryption is transparent and can be run with any operating system.
```

Comment:
```suggestion
     SME uses a single key to encrypt all the memory in the computer. This key
     is ephemeral: it is generated afresh every boot by the CPU. SME must be
     supported in the server's firmware or in the host operating system. If SME
     is enabled in the firmware, the encryption is transparent and can be used with
     any operating system. SME is only supported on AMD processors with the
     AMD Secure feature.
```

Reply:
not sure this is ephemeral, didin't read that in the official doc.

Reply:
I can tell you did; the original version read like it was written by AMD.

I checked, and yes, it is, as far as I can tell.
https://en.wikichip.org/wiki/x86/sme
E.g.

Reply:
yes it is from AMD. This part should be removed from now (PM request we move it probably somwhere else)